browser/src/core/Socket.js

```diff
@@ -1840,19 +1844,13 @@ app.definitions.Socket = L.Class.extend({
 		result.args = args;
 
 		if (this.traceEventRecordingToggle)
```

Comment:
This if condition is redundant

Reply:
True - but hey =) lets get this merged for now.